sdk/lib/crt/wine/heap.c

```diff
@@ -35,51 +60,131 @@
     ((((DWORD_PTR)((char *)ptr + alignment + sizeof(void *) + offset)) & \
       ~(alignment - 1)) - offset))
 
+#define SB_HEAP_ALIGN 16
+
+static HANDLE heap, sb_heap;
 
-typedef int (CDECL *MSVCRT_new_handler_func)(size_t size);
+typedef int (CDECL *MSVCRT_new_handler_func)(MSVCRT_size_t size);
 
 static MSVCRT_new_handler_func MSVCRT_new_handler;
 static int MSVCRT_new_mode;
 
-/* FIXME - According to documentation it should be 8*1024, at runtime it returns 16 */
+/* FIXME - According to documentation it should be 8*1024, at runtime it returns 16 */ 
```

Comment:
![image](https://user-images.githubusercontent.com/578406/213920050-c8eebb4b-d778-4483-8412-844df1794774.png)

Reply:
Potentially said 3rd-party (wine) code had a trailing whitespace?

Reply:
It's okay in this case, which I didn't checked. :sweat_smile: